bundles/org.openhab.binding.ventaair/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,155 @@
+				<option value="45">45 %</option>
+				<option value="50">50 %</option>
+				<option value="55">55 %</option>
+				<option value="60">60 %</option>
+				<option value="65">65 %</option>
+				<option value="70">70 %</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="timer">
+		<item-type>Number</item-type>
+		<label>Timer</label>
+		<description>Timer (0,1,3,5,7,9h)</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Off</option>
+				<option value="1">1 hour</option>
+				<option value="3">3 hours</option>
+				<option value="5">5 hours</option>
+				<option value="7">7 hours</option>
+				<option value="9">9 hours</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="timerTimePassed">
+		<item-type>Number</item-type>
+		<label>Timer Time Passed</label>
+		<description>Time that has passed since set by the Timer (in min)</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="sleepMode">
+		<item-type>Switch</item-type>
+		<label>SleepMode</label>
+		<description>SleepMode</description>
+	</channel-type>
+
+	<channel-type id="boost">
+		<item-type>Switch</item-type>
+		<label>Boost</label>
+		<description>Boost</description>
+	</channel-type>
+
+	<channel-type id="childLock">
+		<item-type>Switch</item-type>
+		<label>ChildLock</label>
+		<description>ChildLock</description>
+	</channel-type>
+
+	<channel-type id="automatic">
+		<item-type>Switch</item-type>
+		<label>Automatic</label>
+		<description>Automatic</description>
+	</channel-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current Temperature</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>Current Humidity</description>
+		<category>Humidity</category>
+		<state pattern="%.1f %%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="waterLevel">
+		<item-type>Number</item-type>
+		<label>WaterLevel</label>
+		<description>Water Level</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">Critical</option>
+				<option value="1">Refill tank</option>
+				<option value="2">OK</option>
+				<option value="3">Full</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fanRPM">
+		<item-type>Number</item-type>
+		<label>Fan RPM</label>
+		<description>Fan RPM</description>
+		<state pattern="%d" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="cleanMode">
+		<item-type>Switch</item-type>
+		<label>Cleaning Mode</label>
+		<description>Device is in cleaning mode (ON)</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="operationTime">
+		<item-type>Number</item-type>
+		<label>Operation Time</label>
+		<description>Operation Time</description>
```

Comment:
This is not enough to know clearly what it does. Example:
Is this how long it will run for next time it is started?
Is this how long it has already been running for?